MonIOWebApp/ClientApp/src/app/models/Budget.ts

```diff
@@ -13,13 +13,20 @@ export class Budget {
 
   public sumExpense:number;
 
-  public monthlyIncome: [BudgetIncome];
+  public monthlyIncome: Array<BudgetIncome>;
 
-  public preTaxExpenses: [BudgetExpense];
+  public preTaxExpenses: Array<BudgetExpense>;
 
-  public taxExpenses: [BudgetExpense];
+  public taxExpenses: Array<BudgetExpense>;
 
-  public regularExpenses: [BudgetExpense];
+  public regularExpenses: Array<BudgetExpense>;
 
-  public spendingCategory: BudgetExpense;
+  public spendingCategory: Array<BudgetExpense>;
+}
+
+export enum BUDGETEXPENSEENUMS {
```

Comment:
I'm assuming this is for Budget_ExpenseCategory Type field...there is a Bill type as well.

Reply:
Should the Bill type be in Budget.ts? I didn't see it in the BudgetViewDTO.

Reply:
I was planning on using these current enum values only for this budget object, but if we need to expand it for other views besides budget I can do that.